src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -66,81 +55,42 @@
     @media screen and (min-width: $pf-global--breakpoint--xl) {
-      max-width: var(--pf-c-modal-box--lg--MaxWidth);
+      --pf-c-modal-box--MaxWidth: var(--pf-c-modal-box--lg--MaxWidth);
     }
   }
 
   // If the last child isn't the footer, then we need to put the footer's bottom padding there
   > :last-child:not(.pf-c-modal-box__footer) {
-    padding-bottom: var(--pf-c-modal-box__footer--PaddingBottom);
+    padding-bottom: var(--pf-c-modal-box__footer--PaddingY);
   }
 }
 
 // Header
 .pf-c-modal-box__header {
   flex: 0 0 auto;
-  padding: var(--pf-c-modal-box__header--PaddingTop) var(--pf-c-modal-box__header--PaddingRight) var(--pf-c-modal-box__header--PaddingBottom) var(--pf-c-modal-box__header--PaddingLeft);
+  padding: var(--pf-c-modal-box__header--PaddingTop) var(--pf-c-modal-box__header--PaddingX) var(--pf-c-modal-box__header--PaddingBottom);
 }
 
-// Header title
-.pf-c-modal-box__header-title {
-  font-size: var(--pf-c-modal-box__header-title--FontSize);
-  font-weight: var(--pf-c-modal-box__header-title--FontWeight);
-  line-height: var(--pf-c-modal-box__header-title--LineHeight);
-
-  // If -webkit-line-clamp is supported
-  @supports (-webkit-line-clamp: var(--pf-c-modal-box--LineClamp)) {
-    // Up to max height var(--pf-c-modal-box--breakpoint-sm--Height): reset display and set single line overflow ellipsis
-    @media screen and (max-height: var(--pf-c-modal-box--breakpoint-sm--Height)) {
-      overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
-    }
-
-    // Between min height var(--pf-c-modal-box--breakpoint-sm--Height) and max height var(--pf-c-modal-box--breakpoint-md--Height): set line clamp
-    @media screen and (min-height: var(--pf-c-modal-box--breakpoint-sm--Height)) and (max-height: var(--pf-c-modal-box--breakpoint-md--Height)) {
-      /* stylelint-disable */
-      display: -webkit-box;
-      overflow: hidden;
-      -webkit-line-clamp: var(--pf-c-modal-box--MaxLineClamp);
-      line-clamp: var(--pf-c-modal-box--MaxLineClamp);
-      -webkit-box-orient: vertical;
-      /* stylelint-enable */
-    }
-
-    // Up to max width $pf-global--breakpoint--sm: limit lines to --pf-c-modal-box--MaxLineClamp
-    @media screen and (max-width: $pf-global--breakpoint--sm) and (min-height: var(--pf-c-modal-box--breakpoint-md--Height)) {
-      /* stylelint-disable */
-      display: -webkit-box;
-      overflow: hidden;
-      -webkit-line-clamp: var(--pf-c-modal-box--MaxLineClamp);
-      line-clamp: var(--pf-c-modal-box--MaxLineClamp);
-      -webkit-box-orient: vertical;
-      /* stylelint-enable */
-    }
-  }
-
-  // If -webkit-line-clamp is not supported, limit text to overflow to single line
-  @supports not (-webkit-line-clamp: var(--pf-c-modal-box--LineClamp)) {
-    @media screen and (max-height: var(--pf-c-modal-box--breakpoint-sm--Height)) {
-      overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
-    }
+// Button close
+.pf-c-modal-box__close {
```

Comment:
I wonder if you can move the class `.pf-c-modal-box__close` to the `.pf-c-button` and remove the wrapping div? it seems unnecessary. Also, can you check with @mceledonia on the button positioning? I would expect the space betweent the close button and the top and right of the modal to be equal. Looking at the [original design PDF](https://drive.google.com/open?id=1WTIXpB9KZOjRWUOHmspxaBJ6RAudvsgF) the spacing was equal, but I wonder if it changed at some point?